help/view-links-on-mobile.md

```diff
@@ -0,0 +1,23 @@
+# View links on mobile
+
```

Comment:
I think this page is more about the configuration, so maybe "Configure how links open", or something like that?

"On mobile" also feels a bit jargony to me as a turn of phrase.

Reply:
I think "Configure how links open" works well.